data_analysis.ipynb

```diff
@@ -4,19 +4,1336 @@
    "cell_type": "markdown",
```

Comment:
<p>fix: the x-axis seems to be displaying years instead of months</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepakkrish212/MATH327-Final/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
The first graph is categorized by month but it is spread through a year